lib/repositories/implementation/genres_repository.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:movie_card/data_models/genre.dart';
+
+import '../interfaces/genres_repository_interface.dart';
+
+class GenresRepository extends GenresRepositoryInterface {
+  static const mockDataGenres = 'mock_data/genres.json';
+  late List<Genre> genresById = [];
+
+  Future<List<Genre>> parseJsonGenres() async {
+    final jsonPath = await rootBundle.loadString(mockDataGenres);
+    final Map<String, dynamic> json = jsonDecode(jsonPath);
+    for (var genre in json['genres']) {
+      genresById.add(Genre(
+        id: genre['id'],
+        name: genre['name'],
+      ));
+    }
```

Comment:
Here you can use the method that is defined on Genre class, instead to use the constructor use `Genre.fromJson`